### PR TITLE
RATIS-1267. Fix ratis examples after supporting streaming

### PR DESCRIPTION
**Pull request by ferhui**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## What is the link to the Apache JIRA

(Please create an issue in ASF JIRA before opening a pull request,
and you need to set the title of the pull request which starts with
the corresponding JIRA issue number. (e.g. RATIS-XXXX. Fix a typo in YYY.)

Please replace this section with the link to the Apache JIRA)

## How was this patch tested?

(Please explain how this patch was tested. Ex: unit tests, manual tests)
(If this patch involves UI changes, please attach a screen-shot; otherwise, remove this)

### CODE REVIEW

**Comment by szetszwo** [comment]
@ferhui , thanks a lot for working on this.  We should make Streaming port optional for these examples since they are not using Streaming.

**Comment by ferhui** [comment]
@szetszwo Thanks for review! 
New commits:

- datastream port is optional
- fix client argument

File: ratis-examples/src/main/java/org/apache/ratis/examples/common/SubCommandBase.java [hunk 1]
```diff
@@ -43,8 +43,12 @@
   public static RaftPeer[] parsePeers(String peers) {
     return Stream.of(peers.split(",")).map(address -> {
       String[] addressParts = address.split(":");
-      return RaftPeer.newBuilder().setId(addressParts[0]).setAddress(addressParts[1] + ":" + addressParts[2])
-          .setDataStreamAddress(addressParts[1] + ":" + addressParts[3]).build();
+      RaftPeer.Builder builder = RaftPeer.newBuilder();
+      builder.setId(addressParts[0]).setAddress(addressParts[1] + ":" + addressParts[2]);
```
Comment (runzhiwang) [inline ratis-examples/src/main/java/org/apache/ratis/examples/common/SubCommandBase.java:47]: maybe we can override parsePeers in DataStream.java

**Review by runzhiwang — APPROVED** [review]
LGTM

**Review by szetszwo — APPROVED** [review]
+1 the change looks good.  Thanks.

**Pull request merged**